application/src/main/java/com/musicforall/web/contact/ContactManagerController.java

```diff
@@ -19,4 +43,21 @@ public ContactManagerController() {
     public String friends(Model model) {
         return "contactManager";
     }
+
+    @RequestMapping(value = "/showUser", method = RequestMethod.GET)
+    public String userpage(@RequestParam("user_id") Integer user_id, Model model) {
+
```

Comment:
create separate class to represent showUser page`s model
fill it with data in separate spring component